experiment_1/scripts/FlexMex1_10/FlexMex1_10_preprocessing.py

```diff
@@ -33,145 +34,12 @@
-        scalars['Parameter'] == 'Transmission_Length_Electricity_Grid'
-    ]
-
-    transmission_capacity = scalars.loc[
-        scalars['Parameter'] == 'Transmission_Capacity_Electricity_Grid'
-    ]
-
-    link['capacity'] = transmission_capacity['Value'].values
-
-    link['loss'] = (
-        transmission_length['Value'].values
-        * 0.01
-        * transmission_loss_per_100km['Value'].values
-        / transmission_capacity['Value'].values
-    )
-
-    link.to_csv(
-        os.path.join(data_preprocessed_path, 'elements', 'link.csv'), index=False,
-    )
-
-
-def update_wind_onshore():
-    wind_onshore = pd.read_csv(os.path.join(data_preprocessed_path, 'elements', 'wind-onshore.csv'))
-
-    scalars_wind_onshore = scalars.loc[
-        scalars['Parameter'] == 'EnergyConversion_Capacity_Electricity_Wind_Onshore'
-    ]
-
-    wind_onshore['name'] = ['-'.join(bus.split('-')[:2] + ['wind-onshore']) for bus in bus_list]
-
-    wind_onshore['carrier'] = 'wind'
-
-    wind_onshore['tech'] = 'onshore'
-
-    wind_onshore['capacity'] = scalars_wind_onshore['Value'].values
-
-    wind_onshore['bus'] = bus_list
-
-    wind_onshore['profile'] = [
-        '-'.join(bus.split('-')[:2] + ['wind-onshore-profile']) for bus in bus_list
-    ]
-
-    wind_onshore.to_csv(
-        os.path.join(data_preprocessed_path, 'elements', 'wind-onshore.csv'), index=False,
-    )
-
-
-def update_wind_offshore():
-    wind_offshore = pd.read_csv(
-        os.path.join(data_preprocessed_path, 'elements', 'wind-offshore.csv')
-    )
-
-    scalars_wind_offshore = scalars.loc[
-        scalars['Parameter'] == 'EnergyConversion_Capacity_Electricity_Wind_Offshore'
-    ]
-
-    wind_offshore['name'] = ['-'.join(bus.split('-')[:2] + ['wind-offshore']) for bus in bus_list]
-
-    wind_offshore['carrier'] = 'wind'
-
-    wind_offshore['tech'] = 'offshore'
-
-    wind_offshore['capacity'] = scalars_wind_offshore['Value'].values
-
-    wind_offshore['bus'] = bus_list
-
-    wind_offshore['profile'] = [
-        '-'.join(bus.split('-')[:2] + ['wind-offshore-profile']) for bus in bus_list
-    ]
-
-    wind_offshore.to_csv(
-        os.path.join(data_preprocessed_path, 'elements', 'wind-offshore.csv'), index=False,
-    )
-
-
-def update_solar_pv():
-    solarpv = pd.read_csv(os.path.join(data_preprocessed_path, 'elements', 'pv.csv'))
-
-    scalars_solarpv = scalars.loc[
-        scalars['Parameter'] == 'EnergyConversion_Capacity_Electricity_Solar_PV'
-    ]
-
-    solarpv['name'] = ['-'.join(bus.split('-')[:2] + ['solarpv']) for bus in bus_list]
-
-    solarpv['carrier'] = 'solar'
-
-    solarpv['tech'] = 'pv'
-
-    solarpv['capacity'] = scalars_solarpv['Value'].values
-
-    solarpv['bus'] = bus_list
-
-    solarpv['profile'] = ['-'.join(bus.split('-')[:2] + ['solar-pv-profile']) for bus in bus_list]
-
-    solarpv.to_csv(
-        os.path.join(data_preprocessed_path, 'elements', 'pv.csv'), index=False,
-    )
-
-
-def combine_volatile_file():
+# not used
```

Comment:
This function seems obsolete. It goes back to a version where all volatile profiles where located in one file. It seems better for overview to keep them separate. This function can be dropped.

Reply:
https://github.com/modex-flexmex/oemo-flex/pull/20/commits/368fa20cc8748096c3928bbcee707ca1af548f67